sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -368,6 +369,7 @@ def infer_output_type(self, input_type):
 
       assert_that(res, equal_to([6, 12, 12, 18, 18, 18]))
 
+  @unittest.skip('https://github.com/apache/beam/issues/23944')
   def test_pardo_large_input(self):
     try:
       utils.check_compiled('apache_beam.coders.coder_impl')
```

Comment:
We should remove this carve out for cython environments if we're going to skip it for every environment (could be done later though if you just want to get this in for test health)

Reply:
I'd rather leave it - when we address this (now sickbayed) test, this could help (even if we can't fix it for non-cythonized envs, if we can make it stable for the rest that's a win). It also might provide investigative context.